diary.cfg

```
@@ -0,0 +1,8 @@
+# Path that holds the journal text files
+diary_dir = /tmp/diary
```

```suggestion
diary_dir = ~/diary
```

I know this is just an example configuration but I think "lose all diary entries on PC restart" is guaranteed to surprise and upset someone.